examples/source/interactivity-dynamic-content/Restrict_Duplicate_Selection.html

```diff
@@ -0,0 +1,92 @@
+    <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <!-- Import the amp-list component to import the list of fruits to select from -->
+    <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+    <!-- Import the amp-selector component so user can choose fruits -->
+    <script async custom-element="amp-selector" src="https://cdn.ampproject.org/v0/amp-selector-0.1.js"></script>
+    <!-- The amp-mustache component is required if using amp-selector with amp-list -->
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  </head>
+  <body>
+    <h2> Restrict Duplicate Selection Example </h2>
+    <!-- ## Implementation -->
+    <!--
+      We use the `<amp-state>` element, list initially empty but will update once user selects fruits, to hold the selected fruits. User selects fruits from '<amp-list>' element fruit_list.
+
+      Every time the user tries to select a fruit the amp-selector element will validate whether or not fruit is already in Fruit_Array_holder.
+      If fruit is not already in the list, amp-selector will change the state of the Fruit_Array_holder with new fruit at end of list. If fruit is in Fruit_Array_holder state stays the same.
+    -->
+    <main class="restrict-duplicate-selection-example">
+    <amp-state id="Fruit_Array_holder" hidden="" aria-hidden="true">
+      <script type="application/json">
+        []
+      </script>
+    </amp-state>
+    <p>
+      Pick your favorite fruites from this list <br> You can only add new fruits to the list, you can't add the same fruit
+      multiple times.
+    </p>
+    <div style="resize: horizontal; overflow: auto;">
+      <p [text]="Fruit_Array_holder">Select fruits!</p>
+    </div>
+    <div id="fruit_list">
+      <amp-list
+        src="/static/samples/json/fruits.json"
+        layout="fixed-height"
+        width="auto"
+        height="900"
+        items="."
+        single-item
+        id="fruit_data_list"
+      >
+        <template type="amp-mustache">
+          <ul>
+            <amp-selector
+              role="tablist"
```

Comment:
Not sure this is the right `role` here, afaik this is used for tabbed UIs. (I might be wrong)